ironfish-cli/src/commands/wallet/import.ts

```diff
@@ -27,6 +29,13 @@ export class ImportCommand extends IronfishCommand {
     },
   ]
 
+  static bech32ToJSON(bech32: string): string {
```

Comment:
This is failable, we should wrap this in a try/catch and return null if it fails. That way we can try and check output for comment below.

Reply:
I see, so instead of 
```
try {
  bech32ToJSON
  } catch {
  ```
  everywhere, move the try/catch into bech32ToJSON, and check for null where bech32ToJSON gets used? Got it